lib/board.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " "," ", " "," ", " "," ", " ", " "]
+    reset!
```

Comment:
I would just call `reset!` here and delegate to that method to create the default empty board. No need to set `@cells` here.